src/parcsr_ls/par_mgr.h

```diff
@@ -7,8 +7,7 @@
 
 #ifndef hypre_ParMGR_DATA_HEADER
 #define hypre_ParMGR_DATA_HEADER
-/*--------------------------------------------------------------------------
- * hypre_ParMGRData
+/*-------------------------------------------------------------------------- * hypre_ParMGRData
```

Comment:
Looks like an accidental deletion of an EOL

Reply:
You're right. I've fixed this. Thank you!